app/assets/javascripts/short_monograph_description.js

```diff
@@ -1,6 +1,6 @@
 jQuery($(document).on('turbolinks:load', function (){
 
-    var minimized_elements = $('.description');
+    var minimized_elements = $('.short_description');
```

Comment:
@gkostin rather than change the class on the div, you should be making your jquery selector more specific -- the monograph `.description` we want to shorten only ever appears in the parent div `.monograph-metadata` -- so if you change this line to ` minimized_elements = $('.monograph-metadata .description');` and change the `index.html.erb` file back to its original `description` class, it should work.